Helper/Config.php

```diff
@@ -1183,6 +1184,22 @@ public function shouldAdjustTaxMismatch($storeId = null)
         return (bool)$this->getAdditionalConfigProperty('adjustTaxMismatch', $storeId);
     }
 
+    /**
+     * Get Bolt additional configuration for Price Mismatch adjustment, stored in the following format:
+     *
+     * {
+     *    "priceFaultTolerance": "1"
+     * }
+     *
+     * @param null $storeId
+     *
+     * @return string
+     */
+    public function getPriceFaultTolerance($storeId = null)
+    {
+        return (int) $this->getAdditionalConfigProperty('priceFaultTolerance', $storeId) ?: Order::MISMATCH_TOLERANCE;
```

Comment:
We need to discuss if we want to continue use this JSON field for additional settings.
We have two other opportunities:
- add hidden setting as we did for local bolt URL
- use feature switch
cc: @matt-thomason